.golangci.yaml

```diff
@@ -0,0 +1,23 @@
+issues:
+  max-same-issues: 0
+linters:
+  enable: # please keep this alphabetized
+    - gofmt
+    - goimports
+    - testifylint
```

Comment:
Running `golangci-lint` locally without a configuration file shows the following linters (as we have been doing in this repository):

```
INFO [lintersdb] Active 6 linters: [errcheck gosimple govet ineffassign staticcheck unused]
```

Should we enable them if they were already enabled by default?